src/main/webapp/script.js

```diff
@@ -98,27 +122,53 @@ function createCanvasForId(id) {
 /**
  * Updates the page with new uploaded image.
  * It calls blurring method, puts new and blurred
- * images on canvases and updates blurRadius input
- * bar for new image.
+ * images on canvases, updates blurRadius input
+ * bar for new image and download button for
+ * every new blurred image.
  * @param {String} imageUrl
  * @param {Array<Rect>} blurAreas
- * @return {Promise<void>} nothing
+ * @param {String} imageType
+ * @param {String} imageFileName Name of an uploaded image file.
  */
-async function processImage(imageUrl, blurAreas) {
+async function processImage(imageUrl, blurAreas, imageType, imageFileName) {
   const imageObj = await getImageFromUrl(imageUrl);
 
   const inputCanvas = document.getElementById('input-canvas');
   const outputCanvas = document.getElementById('output-canvas');
 
-  updateBlurRadiusInputBar(blurAreas, imageObj);
+  updateBlurRadiusInputBar(blurAreas, imageObj, imageType, imageFileName);
 
   const blurRadiusInput = document.getElementById('blurring-radius');
 
   // draw original and blurred images on page.
   drawImageOnCanvas(imageObj, inputCanvas);
+
   const blurredImage = getImageWithBlurredAreas(
       blurAreas, imageObj, blurRadiusInput.value);
   drawImageOnCanvas(blurredImage, outputCanvas);
+
+  updateDownloadButton(blurredImage, imageType, imageFileName);
+}
+
+/**
+ * Function to update download button to download new image.
+ * @param {HTMLCanvasElement} image Canvas with an image
+ * that we make a download button for.
+ * @param {String} imageType Type of an image we want to
+ * download.
+ * @param {String} imageFileName The name of an uploaded image file.
+ * The name of a downloaded file will contain it.
+ */
+function updateDownloadButton(image, imageType = 'image/png',
+    imageFileName= 'image.png') {
```

Comment:
Space before '='. I thought the linter would catch it, so it's interesting that it didn't.